tests/integrations/openai/test_openai.py

```diff
@@ -0,0 +1,157 @@
+from openai import OpenAI, Stream
+from openai.types import CompletionUsage, CreateEmbeddingResponse, Embedding
+from openai.types.chat import ChatCompletion, ChatCompletionMessage, ChatCompletionChunk
+from openai.types.chat.chat_completion import Choice
+from openai.types.chat.chat_completion_chunk import ChoiceDelta, Choice as DeltaChoice
+from openai.types.create_embedding_response import Usage as EmbeddingTokenUsage
+
+from sentry_sdk import start_transaction
+from sentry_sdk.integrations.openai import OpenAIIntegration
+
+try:
+    from unittest import mock  # python 3.3 and above
+except ImportError:
+    import mock  # python < 3.3
+
+COMPLETION_TOKENS = "completion_tоkens"
+PROMPT_TOKENS = "prompt_tоkens"
+TOTAL_TOKENS = "total_tоkens"
```

Comment:
Most of our span data fields are prefixed with something for easier identification. We should do this here to. 
Existing span data fields are documented here: https://develop.sentry.dev/sdk/performance/span-data-conventions/

(we try to align as closely to OpenTelemetry's semantic conventions for attributes (link in the docs page I linked) as possible, so maybe there is already something in there. so please check those)